types/rename/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for rename 1.0
+// Project: https://github.com/popomore/rename
+// Definitions by: Aankhen <https://github.com/Aankhen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.5
```

Comment:
do not think you need the 2.5 label here. this says that the declaration file is only applicable to users with TS version 2.5 or later. the file seems to me to be 2.0 compliant, i.e. no version tag needed. 

Feel free to push an update in a separate PR though.

Reply:
Ah, right… I think I added that to fix some strange behaviour in the tests (on 2.4 and lower, some of the `$ExpectError` tests fail). I’ll remove it along with the failing tests.